### PR TITLE
Add acid resistance spells

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: Mods "Add acid resistance spell"```

#### Purpose of change
Acid resistance spell was requested by KorG before full release of the black dragon lair.

#### Describe the solution
Adds two-tier acid resistance spell, whose scroll recipe is obtained only(per the ``"SECRET"`` flag) by finding Black Dragon: Swamp Ruins. 25% resistance aura upgrades to higher cost, half time 60% resistance aura.

#### Describe alternatives you've considered
Making the 60% resistance spell have equal duration to the normal, axed so the cheaper normal version retains usage for stacking future auras.

#### Testing
Spawned in Black Dragon: Swamp Ruins to see if recipe appears in book. Took spellcraft 2 at chargen to check if SECRET flag works. Crafted it once.
Damage values vs. Black Dragon Breath from adult:
Nekkid w/ greater aura(60%):
image

Nekkid w/ lesser aura(25%):
image

Nekkid w/ both(85%):
image

Nekkid:

image

#### Additional context
Using copy-from for relic data ignores attempted overwriting, which was causing the greater aura to give 100% due to 30% from normal being added to 70% greater when numbers were 30/70.

### CODE REVIEW

File: data/mods/Magiclysm/Spells/classless.json [hunk 1]
```diff
@@ -156,5 +156,50 @@
...
+    "max_range": 0,
+    "min_damage": 1,
+    "max_damage": 1,
+    "effect": "spawn_item",
+    "effect_str": "acid_res_aura_greater",
+    "energy_source": "MANA",
+    "spell_class": "NONE",
+    "flags": [ "CONCENTRATE", "NO_LEGS" ],
+    "difficulty": 7,
+    "max_level": 1,
```
Comment (KorGgenT) [inline data/mods/Magiclysm/Spells/classless.json]: i don't see a reason not to have a max level with duration increase.

File: data/mods/Magiclysm/items/ethereal_items.json [hunk 2]
```diff
@@ -343,5 +343,37 @@
...
+    "relic_data": {
+      "passive_effects": [ { "has": "WORN", "condition": "ALWAYS", "values": [ { "value": "ARMOR_ACID", "multiply": -0.25 } ] } ]
+    }
+  },
+  {
+    "id": "acid_res_aura_greater",
+    "type": "ARMOR",
+    "name": "greater acid resistance aura",
+    "description": "An all-encompassing, invisible layer of protection against acid.",
+    "weight": "0g",
```
Comment (KorGgenT) [inline data/mods/Magiclysm/items/ethereal_items.json]: can't have 0 weight or volume.

File: data/mods/Magiclysm/Spells/classless.json (hunk 1 above)
Comment (LaVeyanFiend) [reply data/mods/Magiclysm/Spells/classless.json]: It's max level 1 so it doesn't look weird casting a level 0 spell forever.

Comment (KorGgenT) [reply data/mods/Magiclysm/Spells/classless.json]: sorry, i was clear. i mean you could have a max level of 5 or 10 and it would be perfectly fine

**Pull request merged**